streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableKTableJoinMerger.java

```diff
@@ -53,8 +90,7 @@ public void enableSendingOldValues() {
         sendOldValues = true;
     }
 
-    private class KTableKTableJoinMergeProcessor<K, V>
-        extends AbstractProcessor<K, Change<V>> {
+    private class KTableKTableJoinMergeProcessor extends AbstractProcessor<K, Change<V>> {
```

Comment:
Why do you remove generics types?

Reply:
It is actually not needed: note that this is a private class that is only used for returning the processor, and hence can completely inherit the generics from its parent `KTableKTableJoinMerger`.